disk_areas.md

```diff
@@ -0,0 +1,78 @@
+
+Abel Disk areas that are relevant for your work at the Veterinary Institute
+
+# Summary :
+
+|directory|path                |what      |characteristics|
+|:--------|:-------------------|:---------|:--------------|
+|`$HOME`|`usit/abel/u1/username`|"home-home" area|500GB - login area- NOT FOR Data Processing -backed-up|
+|**work**|`/work/projects/nn9305k`|common resources (eg. softwares, databases)|10 TB - faster disc, closer to nodes connection|
+|**project**|`/projects/nn9305k`|Project data area (backed-up, raw data, results)|30 TB - slower disc|
```

Comment:
Also project home, for the /projects/nn9305k/home/username area

Reply:
done